src/java/org/apache/cassandra/index/sai/disk/v1/LongArray.java

```diff
@@ -101,27 +98,5 @@ private void open()
     interface Factory
     {
         LongArray open();
-
-        /**
-         * TODO use a different interface for {@link BlockPackedReader}, as {@link LongArray#findTokenRowID(long)} is
-         * not supported by other implementation.
-         *
-         * @param startingIndex minimum index to be used in {@link LongArray#findTokenRowID(long)}.
-         *                      In {@link org.apache.cassandra.index.sai.disk.PostingListRangeIterator}, a segmentRowId
-         *                      is provided and then in {@link OffsetFactory},
-         *                      segment offset is applied to segmentRowId to create sstableRowId which will be used by
-         *                      {@link BlockPackedReader#openTokenReader(long, SSTableQueryContext)}.
-         * @param context shared between indexed columns for the same sstable in a given query
-         * @return token BlockPackedReader
-         */
-        default LongArray openTokenReader(long startingIndex, SSTableQueryContext context)
```

Comment:
I think this is no longer used, so I deleted it to remove `SSTableQueryContext` from `primaryKeyMapFactory#newPerSSTablePrimaryKeyMap`